### DISCUSSION TITLE
Support for Code Annotation in Epub

### DISCUSSION THREAD

**Discussion by vfacta** [quarto-dev/quarto-cli#5975] [Feature Requests]
### Description

Hi! I noticed the code annotation circles are presented just as `Line N`  and are not shown in code block in the epub output, whereas it is fully functional in html and pdf formats. Can the circle feature be supported for a future release of quarto?

Version:
```
quarto --version
1.3.361
```

MWE:

````
---
title: "Untitled"
format: epub
---

```python
print("Hello") # <1>
```

1. This is a test

```c
#include <stdio.h>
 
main()
{
    int a, b;  /* <1> */
 
    a = 2;
    b = 3;

    printf("%d + %d = %d\n", a, b, a+b);
}
```

1. More than one variable can be declared per line

````

`quarto render  test.qmd --to epub`

test.epub:

![Screenshot_2023-06-19_14-56-42](https://github.com/quarto-dev/quarto-cli/assets/120517270/2d029c58-acbf-4761-8b40-f0e954327cd6)

**Comment by mcanouil** [comment]
This is currently by design as explained in <https://quarto.org/docs/authoring/code-annotation.html> with the example of "GitHub Flavored Markdown".
